ydb/core/tx/schemeshard/common/ttl.cpp

```diff
@@ -0,0 +1,19 @@
+#include "ttl.h"
+
+namespace NKikimr::NSchemeShard {
+
+TConclusion<TDuration> GetExpireAfter(const NKikimrSchemeOp::TTTLSettings::TEnabled& settings) {
+    if (settings.TiersSize()) {
+        for (const auto& tier : settings.GetTiers()) {
+            if (tier.HasDelete()) {
+                return TDuration::Seconds(tier.GetApplyAfterSeconds());
+            }
+        }
+        return TConclusionStatus::Fail("TTL settings does not contain DELETE action");
```

Comment:
Is it really correct that `Tiers` collection is invalid if there is no tier with the `Delete` action (it wasn't so in the previous iteration)?

Reply:
For row-oriented tables, only TTL settings with single `Delete` tier are valid, bacause TTL behavior for row-oriented tables is not being changed.